CMakeLists.txt

```diff
@@ -149,15 +149,19 @@ if( MIOPEN_BACKEND STREQUAL "OpenCL")
     find_package( OpenCL REQUIRED )
     find_program(MIOPEN_HIP_COMPILER hcc
             PATH_SUFFIXES bin
-            PATHS /opt/rocm
+            PATHS 
+	        /opt/rocm
+		${CMAKE_INSTALL_PREFIX}
```

Comment:
CMake already searches for in the `CMAKE_INSTALL_PREFIX`, so this isn't needed here.

Reply:
But it fails to.

Reply:
Scratch that, just for clang-offload-bundler.

Reply:
But SWDEV-269156 is about extractkernels isn't it? And this is 4.0.x, so we are not using extractkernels, so why we are looking for it?

Reply:
This is about the symlinks not existing in build dockers. Which means that searching for clang-offload-bundler fails, then the assumption is that we are now in a legacy environment, so MIOpen looks for extractkernel, which also then fails.

Reply:
I see. Usually I ignore that and simply `ln -s /opt/rocm-... /opt/rocm` but this is inconvenient when one works with multiple installations.